test/end-to-end/client.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+const Bluebird = require('bluebird');
+
+const path = require('path');
+const { createManagementClient } = require('../../bin/lib/contentful-client');
+
+const config = {
+  accessToken: process.env.CONTENTFUL_INTEGRATION_MANAGEMENT_TOKEN,
+  application: 'contentful.migration-cli.e2e-test/0.0.0'
```

Comment:
Nice to have a separate useragent thingy here. Maybe we could even `require` the `package.json` and use the version number from there.

Reply:
good idea, will update